okteto.yml

```diff
@@ -1,23 +1,27 @@
 build:
   todo-list:
-    image: okteto.dev/todo-list
-  
+    image: okteto.dev/todo-list:1.0.0
```

Comment:
no need to specify image, we will use `OKTETO_BUILD_TODO_LIST_IMAGE` to refer to the image